src/main/webapp/app/course/learning-paths/services/base-api-http.service.ts

```diff
@@ -66,7 +66,22 @@ export abstract class BaseApiHttpService {
         return await this.request<T>(HttpMethod.Get, url, options);
     }
 
-    protected async post<T>(url: string, body?: any): Promise<T> {
-        return await this.request<T>(HttpMethod.Post, url, { body: body });
+    protected async post<T>(
```

Comment:
Can you add some javadoc above these methods? The parameters get quite complicated...